src/main/scala/com/microsoft/hyperspace/index/IndexConstants.scala

```diff
@@ -94,4 +94,8 @@ object IndexConstants {
   val OPTIMIZE_MODE_QUICK = "quick"
   val OPTIMIZE_MODE_FULL = "full"
   val OPTIMIZE_MODES = Seq(OPTIMIZE_MODE_QUICK, OPTIMIZE_MODE_FULL)
+
+  val UNKNOWN_FILE_ID: Long = -1L
```

Comment:
any reason this is non private whereas below two are?

Reply:
Others are totally internal. This one can show up in index metadata json file. I wanted to make it available for external code in case needed. Let me know if you suggest making it private like others.

Reply:
but this variable is also internal use only, no? (I don't see the relationship between this being public and showing up metadata).

Reply:
Ok, I will make it private for Hyperspace in the next push.